### ISSUE TITLE
Code annotations do not allow horizontal scroll of code blocks as they disappear on click

### ISSUE THREAD

**Issue by pawamoy** [squidfunk/mkdocs-material#3275]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

I'm trying code annotations (nice feature, thanks!).
I've added code blocks in some of them.
The popup is not large enough to display the code block entirely (though the longest line has only 72 characters).
Therefore the code block has a scroll bar at the bottom.
But since the popup disappear on click, it is unscrollable.
Well, actually it is, but I have to click again on the annotation, several times if I want to scroll to the far right.

![image](https://user-images.githubusercontent.com/3999221/144064316-70a436a3-0f68-45d7-be66-7bc346fc17bd.png)
![image](https://user-images.githubusercontent.com/3999221/144064420-12983fa4-81cf-4bf0-bff7-fe57a9f4707f.png)

### Expected behaviour

Allow scrolling code blocks in annotations. EDIT: or selecting text, even 🙂 

### Actual behaviour

Popup disappears when clicking anywhere in it.

### Steps to reproduce

1. Write a code annotation with a relatively large code block in it.
2. Click on the annotation and try to scroll the code block horizontally.

### Package versions

- Python: 3.8.11
- MkDocs: 1.2.3
- Material: 8.0.1

### Configuration

```yaml
site_name: Code Annotations
theme:
  name: material
  features:
  - content.code.annotate
```

### System information

- Operating system: Windows 10
- Browser: Firefox 78.15.0esr

**Comment by squidfunk** [comment]
Oops, the disappear-on-click functionality should only trigger when you click on the annotation marker. That's clearly unintended behavior – thanks for reporting!

**Comment by squidfunk** [comment]
Fixed in 194a08e0f. This worked before but was lost in the refactoring – I forgot to pass the reference element to `getElements`.

**Comment by pawamoy** [comment]
Thanks for the *extremely* quick fix!
